planning/behavior_path_dynamic_avoidance_module/src/scene.cpp

```diff
@@ -414,53 +456,47 @@ BehaviorModuleOutput DynamicAvoidanceModule::planWaitingApproval()
   return out;
 }
 
-bool DynamicAvoidanceModule::isLabelTargetObstacle(const uint8_t label) const
+ObjectType DynamicAvoidanceModule::getLabelAsTargetObstacle(const uint8_t label) const
 {
   using autoware_auto_perception_msgs::msg::ObjectClassification;
 
   if (label == ObjectClassification::CAR && parameters_->avoid_car) {
-    return true;
+    return ObjectType::REGULATED;
   }
   if (label == ObjectClassification::TRUCK && parameters_->avoid_truck) {
-    return true;
+    return ObjectType::REGULATED;
   }
   if (label == ObjectClassification::BUS && parameters_->avoid_bus) {
-    return true;
+    return ObjectType::REGULATED;
   }
   if (label == ObjectClassification::TRAILER && parameters_->avoid_trailer) {
-    return true;
+    return ObjectType::REGULATED;
   }
   if (label == ObjectClassification::UNKNOWN && parameters_->avoid_unknown) {
-    return true;
+    return ObjectType::UNREGULATED;
   }
   if (label == ObjectClassification::BICYCLE && parameters_->avoid_bicycle) {
-    return true;
+    return ObjectType::UNREGULATED;
   }
   if (label == ObjectClassification::MOTORCYCLE && parameters_->avoid_motorcycle) {
-    return true;
+    return ObjectType::REGULATED;
   }
   if (label == ObjectClassification::PEDESTRIAN && parameters_->avoid_pedestrian) {
-    return true;
+    return ObjectType::UNREGULATED;
   }
-  return false;
+  return ObjectType::OUT_OF_SCOPE;
 }
 
-void DynamicAvoidanceModule::updateTargetObjects()
+void DynamicAvoidanceModule::registerRegulatedObjects(
+  const std::vector<DynamicAvoidanceObject> & prev_objects)
 {
   const auto input_path = getPreviousModuleOutput().path;
   const auto & predicted_objects = planner_data_->dynamic_object->objects;
 
-  const auto input_ref_path_points = getPreviousModuleOutput().reference_path.points;
-  const auto prev_objects = target_objects_manager_.getValidObjects();
-
-  updateRefPathBeforeLaneChange(input_ref_path_points);
+  // const auto input_ref_path_points = getPreviousModuleOutput().reference_path.points;
 
-  const auto & ego_pose = getEgoPose();
-  const double ego_vel = getEgoSpeed();
-  const auto ego_lat_feasible_paths = generateLateralFeasiblePaths(ego_pose, ego_vel);
+  // updateRefPathBeforeLaneChange(input_ref_path_points);
```

Comment:
Should these comment out removed?

Reply:
I wondered that we should remove these or not.
Now I'll remove these